cmd/goProbe/config/config.go

```diff
@@ -8,39 +8,65 @@
 //
 /////////////////////////////////////////////////////////////////////////////////
 
-// Package for parsing goprobe config files.
+// Package config is for parsing goprobe config files.
 package config
 
 import (
 	"encoding/json"
 	"fmt"
-	"io/ioutil"
 	"os"
+	"sync"
 
 	"github.com/els0r/goProbe/pkg/capture"
 )
 
+// Mutex exposes a global lock for the configuration
+var Mutex sync.Mutex
```

Comment:
Afaik it is best practice to add the Mutex as anonymous member to the config struct itself, then you can do cfg.Lock() and cfg.Unlock(), which is cleaner that a global variable...